lib/ui/cards/student_info/student_info_card.dart

```diff
@@ -56,28 +58,39 @@ class _StudentInfoCardState extends State<StudentInfoCard> {
     var tokenQueryString =
         "token=" + '${_userDataProvider.authenticationModel.accessToken}';
     var url = _url + "?" + tokenQueryString;
-    return Column(
-      children: <Widget>[
-        Flexible(
-            child: WebView(
-          javascriptMode: JavascriptMode.unrestricted,
-          initialUrl: url,
-          onWebViewCreated: (controller) {
-            _webViewController = controller;
-          },
-          javascriptChannels: <JavascriptChannel>[
-            _printJavascriptChannel(context),
-          ].toSet(),
-        )),
-      ],
-    );
+    print(url);
+    return Container(
+        height: _contentHeight,
+        child: WebView(
+            javascriptMode: JavascriptMode.unrestricted,
+            initialUrl: url,
+            onWebViewCreated: (controller) {
+              _webViewController = controller;
+            },
+            javascriptChannels: <JavascriptChannel>[
+              _printJavascriptChannel(context),
+            ].toSet(),
+            onPageFinished: (some) async {
+              double height = double.parse(await _webViewController
```

Comment:
would recommend moving all this logic to a separate function to keep extra logic out of buildCardContent